### DISCUSSION TITLE
help regarding contribution

### DISCUSSION THREAD

**Discussion by ayansaha581** [wazuh/wazuh#27393] [Q&A] [answered]
I would like add some SCA rules for MacOS, the master branch doesn't seem updated, confused as to what should take as base and add my changes ?

**Comment by Damian-Mangold** [comment]
Hi @ayansaha581 Let me consult with the SCA team and I will get back to you as soon as possible.

**Reply by Damian-Mangold** [reply, marked as answer]
Hi @ayansaha581. You can use the master branch as your base to add your SCA rules for macOS. If there are adjustments needed later, we can address them as part of the process. Once you've made your changes, please share the link to your pull request (PR) so we can review it and provide feedback. Let us know if you encounter any issues along the way!

**Reply by ayansaha581** [reply]
Thanks @Damian-Mangold ! I have created a PR for the same - https://github.com/wazuh/wazuh/pull/27463 . I will soon be leaving the company on behalf of which I am submitting this PR (17th Jan is my Last Date). So if it can be merged before that, that would be nice!

**Reply by Damian-Mangold** [reply]
Perfect. I have already alerted the team in charge of the review. If you have any questions, please continue from the PR so that everything is concentrated in one place.

**Comment by sebasfalcone** [comment]
Discussion moved to https://github.com/wazuh/wazuh/pull/27463
